src/wallet/scriptpubkeyman.h

```diff
@@ -218,6 +218,8 @@ class ScriptPubKeyMan
 
     /** Keypool has new keys */
     boost::signals2::signal<void ()> NotifyCanGetAddressesChanged;
+
+    virtual bool HasWalletDescriptor(const WalletDescriptor& desc) const { return false; }
```

Comment:
I don't really like having this be part of `ScriptPubKeyMan`. I feel like this interface shouldn't really care about the internal details of its sub classes, so this is kind of out of place.